sim/midas/src/main/scala/midas/passes/AutoCounterCoverTransform.scala

```diff
@@ -225,17 +226,24 @@ class AutoCounterTransform extends Transform with AutoCounterConsts {
       case o => remainingAnnos += o
     }
 
+    println(s"[AutoCounter] There are ${counterAnnos.length} counterAnnos available for selection in the following modules:")
+    counterAnnos.map(_.target.module).distinct.foreach({ i => println(s"  ${i}") })
+
     //extract the module names from the methods mentioned previously
     val covermodulesnames = moduleAnnos.map(_.target.module).distinct
+    println("[AutoCounter] covermodulesnames are:")
```

Comment:
I'm generally in favor of giving a more "human readable" descriptions in log lines. i.e. something like "Selected Modules for cover-function based AutoCounter annotation" rather than "covermodulesnames"

Reply:
Good point @alonamid , what do you think of :
```suggestion
    println("[AutoCounter] selected modules for cover-function based annotation:")
```

Reply:
works for me